news/morphsqueeze.rst

```diff
@@ -0,0 +1,26 @@
+**Added:**
+
+* Squeeze morph, test with sine wave
+
+**Changed:**
+
+* Added cubic and quadratic terms, as well as
```

Comment:
none of these are needed.  We just need one news item per PR basically, not a history of everything that happened on the way.  So what is needed is, under Added, something like "Squeeze morph that carries out a non-linear stretch to the independent variable axis."